### PR TITLE
Added some text to the new dev docs

### PR DESCRIPTION
**Pull request by johndeange**
## What changed

Filled in some of the sections for #2141

### CODE REVIEW

**Review by stevtek — APPROVED** [review]

**Review by jonnalley — APPROVED** [review]

**Review by fpigeonjr — APPROVED** [review]
⭐

File: docs/developers/onboarding.md [hunk 1]
```diff
@@ -18,12 +18,65 @@ Brief description of what the project is about, its goals, and how it fits into
...
+* [Marshmallow](https://marshmallow.readthedocs.io/en/stable/)
+* [Pytest](https://docs.pytest.org/en/6.2.x/)
+* [Flask JWT Extended](https://flask-jwt-extended.readthedocs.io/en/stable/)
+* [PostgreSQL](https://www.postgresql.org/)
+
 ## DevOps Practices
 
+The project uses GitHub Actions for CI/CD.  The workflows are defined in the `.github/workflows` directory.
+The IaC for the project is defined using Terraform.
+The project is deployed to Azure using Container Instances.
```
Comment (devops-chris) [inline docs/developers/onboarding.md:62]: Maybe it's not a significant detail to the audience of this document, but we are actually using Azure Container App Service vs. Azure Container Instances

**Comment by fpigeonjr** [comment]
- completed some missing sections
- added to FE section adding overviews and tutorials

File: docs/developers/onboarding.md [hunk 2]
```diff
@@ -14,16 +14,104 @@ Welcome aboard! We're excited to have you as part of our team. This guide is des
...
+OPS will empower managers and staff with the visibility they need into OPRE’s research, evaluation, and data activities, as well as streamline budget and administrative tasks. With this system, it will be easy for OPRE to plan and manage projects so they can spend less time managing the work and more time building evidence to improve the lives of children and families.
+
+- read more on the [wiki](https://github.com/HHS/OPRE-OPS/wiki/01-Background-and-introduction)
 
 ## Repository Setup
 
+The OPS project is hosted in two separate repositories.  Both repositories are hosted on GitHub.
+
+- [OPRE-OPS](https://github.com/HHS/OPRE-OPS) - The primary application.
+- [OPRE-OPS-Data](https://github.com/HHS/OPRE-OPS-Data) - IaC and scripts for the data migration.
```
Comment (fpigeonjr) [inline docs/developers/onboarding.md:28]: IaC? maybe we should spell this out?

File: docs/developers/onboarding.md (hunk 1 above)
Comment (jonnalley) [reply docs/developers/onboarding.md:95]: well, they are different things for sure

**Pull request merged**